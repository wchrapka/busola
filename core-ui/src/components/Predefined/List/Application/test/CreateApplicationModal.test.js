import { createApplicationInput } from '../CreateApplicationModal';

describe('createApplicationInput', () => {
  it('forms valid application input', () => {
    const input = {
      name: 'test-app-name',
      description: 'test-description',
      labels: { a: 'b', c: 'd', d: 'e', e: 'f' },
    };
    expect(createApplicationInput(input)).toMatchSnapshot();
  });
});
