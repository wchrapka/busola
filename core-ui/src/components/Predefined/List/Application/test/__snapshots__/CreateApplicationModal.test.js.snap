// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createApplicationInput forms valid application input 1`] = `
Object {
  "apiVersion": "applicationconnector.kyma-project.io/v1alpha1",
  "kind": "Application",
  "metadata": Object {
    "labels": Object {
      "a": "b",
      "c": "d",
      "d": "e",
      "e": "f",
    },
    "name": "test-app-name",
  },
  "spec": Object {
    "accessLabel": "test-app-name",
    "description": "test-description",
    "services": Array [],
  },
}
`;
